RecoTracker/FinalTrackSelectors/python/MergeTrackCollections_cff.py

```diff
@@ -31,6 +31,9 @@
 duplicateTrackClassifier.mva.min3DLayers = [0,0,0]
 duplicateTrackClassifier.mva.maxLostLayers = [99,99,99]
 
+from Configuration.ProcessModifiers.run4_PixelCPEGeneric_cff import run4_PixelCPEGeneric
+run4_PixelCPEGeneric.toModify(duplicateTrackCandidates, ttrhBuilderName = "WithTrackAngle") # FIXME
```

Comment:
what is the purpose of "FIXME" here?

Reply:
copy-paste error, from undoing https://github.com/cms-sw/cmssw/pull/28448/commits/aeffc457daf6341d546449b19535f0f557a9d772. 
@duartej please clean this.

Reply:
done